README.md

```diff
@@ -1,24 +1,57 @@
-# README
+# Budget-App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- Is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby on Rails
 
-* System dependencies
+### Video Link
+- Here is a link to the video explaining the project [link](https://www.loom.com/share/bcb5bbd8f88848afb995e0b694c26583)
 
-* Configuration
+## Live Demo
 
-* Database creation
+No live demo available
```

Comment:
You have two live demo sections in your README me. Since you have another section that has the live version link, there's no need to have this section. Kindly remove it.